engine/install.go

```diff
@@ -158,6 +159,12 @@ poller:
 		fmt.Sprintf("DRONE_RUNNER_PRIVILEGED_IMAGES=%s", i.runner.Privileged),
 	)
 
+	if len(i.labels) > 0 {
+		envs = append(envs,
+			fmt.Sprintf("DRONE_RUNNER_LABELS=%s", strings.Join(i.labels, ",")),
```

Comment:
Not 100% sure here, I think with the `map` we could just do `fmt.Sprintf("DRONE_RUNNER_LABELS=%s", i.labels),` in the big `append` above

Reply:
if you have `map[string]string{"foo":"bar"}` it would print `map[foo:bar]`

Reply:
Thanks, I hadn't tested this bit, completely new to `go`. Sorry for the bad advice!